ranger/container/tags.py

```diff
@@ -21,12 +21,15 @@ def __init__(self, filename):
 
         if isdir(dirname(self._filename)) and not exists(self._filename):
             open(self._filename, 'w')
-
+        
         self.sync()
 
     def __contains__(self, item):
         return item in self.tags
 
+    def __getitem__(self, fobj):
```

Comment:
This sounds like a *bad* idea. Implement a way to *query* the tags object sure. Let's not give total control over this class's only important data structure to whomever.

Reply:
I hope I understood you correctly and changed it to `return self.tags[path]`